.github/ISSUE_TEMPLATE/models.md

```diff
@@ -35,13 +35,11 @@ While the default shape for new components is a circle, each component should be
 
 ### Instructions for Relationships
 1. Identify the relationship and any specific constraints to be enforced between the two specific components, their models or potentially other components, models, or environmental considerations.
-1. Propose a specific visual representation for the relationship. Visual representation examples: [Hierarchical](https://docs.meshery.io/assets/img/meshmodel/relationships/hierarchical_relationship.png), [Sibling](https://docs.meshery.io/assets/img/meshmodel/relationships/sibling_relationship.png), [Binding](https://docs.meshery.io/assets/img/meshmodel/relationships/binding_relationship.png), [Edge](https://docs.meshery.io/assets/img/meshmodel/relationships/mount_edge_relationship.png)
```

Comment:
@leecalcote are you suggesting removing all steps in ```Instructions for Relationships``` and Simply add See example [here](https://docs.meshery.io/project/contributing/contributing-models). As contrib models docs contains the same content mentioned here. 
Right?